index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header">
+    <div class="header-content">
+      <div class="logo">
+        <a href="#">My Logo</a>
+      </div>
+      <img class="hamburger" src="images/menu.svg" alt="menu">
+    </div>
+  </header>
+  <section class="section-text">
```

Comment:
- [ ] please try to put everything in your headline inside the section container.

Reply:
Hello @tufoinnkuo10, did you mean to do this? Put my header inside the section container?

```
<section class="section-text">
    <header class="header">
      <div class="header-content">
        <div class="logo">
          <a href="#">Daniel</a>
        </div>
          <nav>
            <img class="hamburger" src="images/menu.svg" alt="menu">
          </nav>
      </div>
    </header>
    <div class="section-title">
      <h1>I'm Daniel</h1> 
        <h2>Glad to see you!</h2>
    </div>
    <div class="section-content">
      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
    </div>
</section>
```